test/functional/test_framework/script.py

```diff
@@ -904,41 +904,41 @@ def construct_pkhasholder(self, key, data, n): #ECPubKey, 20B, int
         self._set_miniscript(miniscript.and_v(v_c_pk_node, miniscript.and_v(v_hash_node, older_node)))
         self.desc = TapLeaf._desc_serializer('pkhasholder', key.get_bytes().hex(), data.hex(),str(n))
 
-    def construct_csa(self, k, *args): #int, ECPubKey ...
-        keys_data = [key.get_bytes() for key in args]
-        thresh_csa_node = miniscript.thresh_csa(k, *keys_data)
+    def construct_csa(self, n, pkv):
```

Comment:
Optech house style is to refer to threshold signatures as _k-of-n_ (n is the number of pubkeys and k is the number of signatures required). This is because when spoken 'm-of-n' sounds very similar to 'n-of-n'. (https://github.com/bitcoinops/bitcoinops.github.io/blob/c32a183d86e65487389ebb8b44c5590b57fe8014/STYLE.md#spelling)

For what it's worth, sipa also uses k-of-n when referring to threshold signatures.

Can we change these function arguments to be _k_ throughout this PR?